src/tools/miri/tests/fail/upvars/deref-in-pattern.rs

```diff
@@ -0,0 +1,19 @@
+// This test serves to document the change in semantics introduced by #138961.
+//
+// A corollary of partial-pattern.rs: while the tuple access showcases the
+// utility, it is actually the dereference performed by the pattern that
```

Comment:
The utility of what?

Reply:
The utility of the semantics, i.e. why the closure might want to capture the deref'ed value.